android/app/src/main/java/com/dismoi/scout/accessibility/browser/Chrome.kt

```diff
@@ -0,0 +1,46 @@
+package com.dismoi.scout.accessibility.browser
+
+import android.os.Build
+import android.util.Log
+import android.view.accessibility.AccessibilityNodeInfo
+import androidx.annotation.RequiresApi
+
+class Chrome(): Browser() {
```

Comment:
Why not merge this by extanding SupportedBrowserConfig ?

Reply:
What do you mean? Example?

Reply:
I could do a PR … Is you refacto done ? (I don’t want to interrupt or generate conficts)

Reply:
Sure! I'm done. But please test it well before PR: here are the scenarios > https://github.com/dis-moi/mobile_poc/blob/master/.github/ISSUE_TEMPLATE/test-scenarii.md